server/index.js

```diff
@@ -8,6 +8,7 @@ const app = express();
 const path = require('node:path');
 const publicPath = path.join(__dirname, 'public');
 const jwt = require('jsonwebtoken');
+const jwtDecode = require('jwt-decode');
```

Comment:
On the server side, we need to use the `'jsonwebtoken'` library to `.verify()` the token.

`jwtDecode` doesn't make sure that the signature is valid, so it leaves the back end susceptible to forged tokens